chapters/4-acquisition.tex

```diff
@@ -807,6 +798,23 @@ \subsection{Backing up original data}
 If you remain lucky, you will never have to access your ``master'' or ``golden master'' copies --
 you just want to know it is there, safe, if you one day end up needing it.
 
-Once the raw data is securely stored and backed up,
-it can be transformed into your final analysis dataset
-through the steps described in the next chapter.
+\section{Looking ahead}
+
+Once the raw data is completely transferred, securely stored and backed up,
+the data acquisition stage, summarized in figure \ref{fig:acquisition}, is complete.
```

Comment:
```suggestion
then the data acquisition stage, summarized in figure \ref{fig:acquisition}, is complete.
```
I had to read this sentence a couple of times. I didnt get it until I added a "then" in my mind